ci-operator/step-registry/gather/snc/gather-snc-ref.yaml

```diff
@@ -0,0 +1,27 @@
+ref:
+  as: gather-snc
+  from: snc
+  commands: gather-snc-commands.sh
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  env:
```

Comment:
These are intended to expose parameters to the users that execute this step. Which of these are parameters that should be set by the user at runtime?

Reply:
@stevekuznetsov as of now, these parameter make use of default value so in the runtime user doesn't need to set anything but in future if any of parameter need to pass by step then these can be used.